Support Remote Health docs fixes during ROSA review (#63644)
openshift/openshift-docs: modules/disabling-insights-advisor-recommendations.adoc

.Procedure

. Navigate to *Advisor* -> *Recommendations* on {cluster-manager-url}.

> . Optional: Use the *Clusters Impacted* and *Status* filters, as needed.

Reviewer: ```suggestion
. Optional: Use the *Clusters Impacted* and *Status* filters as needed.
```

. Disable an alert by using one of the following methods: 
+
* To disable an alert:

> .. Click the *Options* menu {kebab} for that alert and then click *Disable recommendation*.

Reviewer: ```suggestion
.. Click the *Options* menu {kebab} for that alert, and then click *Disable recommendation*.
```

.. Enter a justification note and click *Save*.
+
* To view the clusters affected by this alert before disabling the alert:
.. Click the name of the recommendation to disable. You are directed to the single recommendation page.
.. Review the list of clusters in the *Affected clusters* section.

> .. Disable the alert for all of your clusters by clicking *Actions* -> *Disable recommendation*.

Reviewer: ```suggestion
.. Click *Actions* -> *Disable recommendation* to disable the alert for all of your clusters.
```
